eng/pipelines/coreclr/interpreter.yml

```diff
@@ -73,8 +73,6 @@ extends:
           - linux_arm64
           - windows_x64
           - windows_arm64
-          - osx_x64
```

Comment:
Why have you removed the osx_arm64? It is actually the most important configuration.

Reply:
That was just temporary. I was trying to reduce the load on the servers as I did this big runs to find out if the CI could be coerced into getting good data.